### ISSUE TITLE
Crash when picking up a circuit

### ISSUE THREAD

**Issue by kaytrim** [amadornes/SCM-Localization-Issues#310]
this is a circuit I made and was breaking with a tinkers pick.  http://pastebin.com/Ui8Wxaj5

**Comment by kaytrim** [comment]
This happened twice now with the same pick from Tinkers but different circuit plates. Playing on SkyFactory 3.0.4

**Comment by shabble** [comment]
[crash-2017-01-14_07.34.20-client.txt](https://github.com/amadornes/SCM-Localization-Issues/files/705817/crash-2017-01-14_07.34.20-client.txt)
What appears to be an identical crash (happened at least twice now, but not consistently - sometimes breaking circuit plates is fine, other times it crashes). Tool used to break is both a tinkers pick and an Actually Additions drill. Both had silk touch.

Crash log is attached.

Same as Kaytrim, running SkyFactory 3.0.4 beta which looks to be using SCM 1.0.4

Any other info that would be useful? I'll update if I can reproduce more consistently.

**Comment by keldor314** [comment]
Yup.  Getting the same crash here too.  Tinker's pickaxe also.

It also didn't give me my circuit back after I restarted.

**Comment by desht** [comment]
It's not just Tinker's tools - I've had the same crash with an EnderIO dark pick, and also with a bare hand.

**Comment by mrtomwolf** [comment]
Same here, though it's a bit disheartening to not see a single reply by the dev on issues here.

**Comment by ShinyPhoenix** [comment]
I am experiencing the same issue on SkyFactory 3 3.0.6. 

Crash report: http://pastebin.com/nmCeA6hs

Disabling Waila _solves_ the issue.

**Comment by desht** [comment]
When you say "disabling", do you mean remove the Waila mod entirely or just turn off the info box?

**Comment by ShinyPhoenix** [comment]
No, @desht - remove the mod (Hwyla-1.8.10-B24_1.10.2, Waila port for Minecraft 1.10.2 - SkyFactory 3 3.0.6) completely.

**Comment by desht** [comment]
Thanks, will try it later.  I guess The One Probe can always fill in... shame it's not in the pack by default, though.

**Comment by TheSomnambulist** [comment]
I am getting the same thing, but only when i break circuits in quick succession.  I have a ticon silk touch pick. I can post it's materials/modifiers/stats if needed, but based on the other comments I don't think i need too.

**Comment by thelapisfox** [comment]
Getting this same issue on the Age of Engineering mod pack. It doesn't matter what I break the circuit with. Also, I just started using SCM so I am using the pre-made circuits (specifically, the comparator). Removing HWYLA does not stop this from occurring.
